readme.md

```diff
@@ -450,3 +461,36 @@ body, .foo {
   [RULE_END]  
 ]
 ```
+
+#### JavaScript Function Value
```

Comment:
It looks like that's implied here since the example below returns an array of rules, instead of a single tuple.

But it should indeed say that the result of a function can be an entire ISTF array.

Also, it's highly unlikely that we'll have control over all function's values. We're implying here that all results of such interpolations need to be preparsed (think babel plugin), so in the case of SC every function return value would need to be wrapped in a css tagged template string helper so that it's targetable in a babel plugin.

Since this is impractical we should add that a function can also just return any value that has an implicit interpretation.

Now, this leads me to selectors: A function might also return an interpolation for a selector. So we should introduce a second function type, that is part of a selector.

This might require us to break the selector tuple apart though.